frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -49,7 +49,7 @@ pods:
           path: "container-path"
           type: {{MASTER_NODE_DISK_TYPE}}
           size: {{MASTER_NODE_DISK}}
-        cmd: "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/) && ./elastic-scheduler/java-dns-ttl.sh && ./elastic-scheduler/install-plugins.sh && ./elastic-scheduler/setup-java-policy.sh &&  ./bootstrap && exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch"
+        cmd: "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/) && ./elastic-scheduler/java-dns-ttl.sh && ./elastic-scheduler/install-plugins.sh && ./elastic-scheduler/setup-keystore-bootstrap-password.sh && ./elastic-scheduler/setup-java-policy.sh && ./bootstrap && exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch"
```

Comment:
While we're at it, we could switch to a multi-line command (see HDFS's `svc.yml` as an example.

Adding `set -e` could also be useful to not hide errors.

Reply:
Out of scope fro this PR: It seems as if we keep adding scripts to the list (and these need to be added for **every** `cmd`). Would it make sense to have a single `init.sh` (for example) script that calls the other scripts instead?

Reply:
> Would it make sense to have a single init.sh (for example) script that calls the other scripts instead?

I thought about doing that but ultimately decided to just follow what was already being done for consistency. I'm +1 on the idea. For this PR I'll push at least a change making this a multi-line command.

Reply:
Yes, multi line commands with `set -e` are the new hotness!

Reply:
https://github.com/mesosphere/dcos-commons/pull/2078/commits/db06f43c0545825a20e3ee0353a5b2ca7367e04f